plugin/ranger.vim

```diff
@@ -48,10 +48,10 @@ if has('nvim')
       endif
       try
         if filereadable(s:choice_file_path)
-          exec system('sed -ie "s/ /\\\ /g" ' . s:choice_file_path)
-          exec 'argadd ' . system('cat ' . s:choice_file_path . ' | tr "\\n" " "')
-          exec self.edit_cmd . system('head -n1 ' . s:choice_file_path)
-          call system('rm ' . s:choice_file_path)
+          for f in readfile(s:choice_file_path)
+            exec 'edit '. f
```

Comment:
please use the params `self.edit_cmd` instead of 'edit ' to keep the "open the file in a new tab" feature working